packages/esm-patient-chart-app/src/patient-flags/patient-flags-list.component.tsx

```diff
@@ -0,0 +1,193 @@
+
+  const searchResults = useMemo(() => {
+    if (!isEmpty(searchTerm)) {
+      return sortedRows.filter((f) => f.flag.display.toLowerCase().search(searchTerm.toLowerCase()) !== -1);
+    } else {
+      return sortedRows;
+    }
+  }, [searchTerm, sortedRows]);
+
+  const handleSearch = useMemo(() => debounce((searchTerm) => setSearchTerm(searchTerm), 300), [setSearchTerm]);
+
+  const handleSortByChange = ({ selectedItem }: { selectedItem: string }) => setSortBy(selectedItem);
+
+  const handleEnableFlag = async (flagUuid) => {
+    setEnabling(true);
+    const res = await enablePatientFlag(flagUuid);
+
+    if (res.status === 200) {
+      mutate();
+      showToast({
+        critical: true,
+        kind: 'success',
+        description: t('flagEnabled', 'Patient flag successfully enabled'),
+        title: t('enablingFlag', 'Enabling flag'),
+      });
+
+      setEnabling(false);
+    } else {
+      showNotification({
+        critical: true,
+        kind: 'error',
+        description: t('errorEnabling', 'There was a problem enabling the flag'),
+        title: t('enablingFlag', 'Enabling flag'),
+      });
+    }
+  };
+
+  const handleDisableFlag = async (flagUuid) => {
+    setDisabling(true);
+    const res = await disablePatientFlag(flagUuid);
+
+    if (res.status === 204) {
+      mutate();
+      showToast({
+        critical: true,
+        kind: 'success',
+        description: t('flagDisabled', 'Patient flag successfully disabled'),
+        title: t('disablingFlag', 'Disabling flag'),
+      });
+
+      setDisabling(false);
+    } else {
+      showNotification({
+        critical: true,
+        kind: 'error',
+        description: t('errorDisabling', 'There was a problem disabling the flag'),
+        title: t('disablingFlag', 'Disabling flag'),
+      });
+    }
+  };
+
+  if (isLoadingFlags) {
+    return <Loader />;
+  }
+  if (flagLoadingError) {
+    return <div>{flagLoadingError.message}</div>;
+  }
+  return (
+    <Form className={styles.formWrapper}>
+      <Stack gap={4}>
+        <div className={styles.listWrapper}>
+          <Search
+            size="lg"
+            light={isTablet}
+            value={searchTerm}
+            placeholder={t('searchForAFlag', 'Search for a flag')}
+            labelText={t('searchForAFlag', 'Search for a flag')}
+            onChange={(e) => handleSearch(e.target.value)}
+          />
+          <div className={styles.flagsHeaderInfo}>
+            <div>
+              {searchResults.length} {searchResults.length > 1 ? 'flags' : 'flag'}
+            </div>
+            <div>
+              <Dropdown
+                className={styles.sortDropdown}
+                id="sortBy"
+                initialSelectedItem={'A - Z'}
+                label=""
+                titleText={t('sortBy', 'Sort by') + ':'}
+                type="inline"
+                items={['A - Z', t('activeFirst', 'Active first'), t('retiredFirst', 'Retired first')]}
```

Comment:
Let's translate A-Z as well or we can rename it to "Alphabetically", because not all languages have A-Z alphabets